lib/protobuf/nats/server.rb

```diff
@@ -80,6 +80,26 @@ def enqueue_request(request_data, reply_id)
         was_enqueued
       end
 
+      def do_not_subscribe_to_includes?(subscription_key)
+        return false if ::Protobuf::Nats.config.server_subscription_key_do_not_subscribe_to_when_includes_any_of.nil?
```

Comment:
You can probably remove the first two predicates since we end up checking `respond_to?` anyway. This code path should be used very infrequently anyway. Same thing for `only_subscribe_to_includes?`.